### PR TITLE
implement minor onboarding ui improvements

### PR DESCRIPTION
**Pull request by jpangelle**
### Description

This pull request implements some minor improvements to the onboarding ui.

### Approvals

- [ ] Dev

### CODE REVIEW

File: centrifuge-app/src/pages/Onboarding/EmailVerified.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ export const EmailVerified: React.FC = () => {
   const { error, data } = useVerifyEmail()
 
   return (
-    <Flex backgroundColor="backgroundSecondary" minHeight="100vh" flexDirection="column" textAlign="center">
+    <Flex backgroundColor="#f9f9f9" minHeight="100vh" flexDirection="column" textAlign="center">
```
Comment (jpangelle) [inline centrifuge-app/src/pages/Onboarding/EmailVerified.tsx:15]: changed background based on Araya's advice

File: centrifuge-app/src/pages/Onboarding/EmailVerified.tsx [hunk 2]
```diff
@@ -29,6 +29,7 @@ export const EmailVerified: React.FC = () => {
         backgroundColor="backgroundPrimary"
         alignItems="flex-start"
         gridTemplateColumns="1fr"
+        maxWidth="1200px"
```
Comment (jpangelle) [inline centrifuge-app/src/pages/Onboarding/EmailVerified.tsx:32]: also based on Araya's advice

File: onboarding-api/src/controllers/user/updateInvestorStatus.ts [hunk 3]
```diff
@@ -38,15 +38,15 @@ export const updateInvestorStatusController = async (
         user.investorType === 'individual' &&
         !user.countryOfCitizenship?.startsWith('us')
       ) {
-        return true
+        return false
```
Comment (jpangelle) [inline onboarding-api/src/controllers/user/updateInvestorStatus.ts:41]: logic was inverted

File: onboarding-api/src/controllers/user/updateInvestorStatus.ts [hunk 4]
```diff
@@ -36,16 +36,12 @@ export const updateInvestorStatusController = async (
       if (
         name === 'verifyAccreditation' &&
         user.investorType === 'individual' &&
-        !user.countryOfCitizenship?.startsWith('us')
+        user.countryOfCitizenship?.startsWith('us')
```
Comment (jpangelle) [inline onboarding-api/src/controllers/user/updateInvestorStatus.ts:39]: logic was inverted

File: centrifuge-app/src/pages/Onboarding/EmailVerified.tsx (hunk 1 above)
Comment (sophialittlejohn) [reply centrifuge-app/src/pages/Onboarding/EmailVerified.tsx:15]: can we add this to the theme for it looks decent for altair as well?

Comment (sophialittlejohn) [reply centrifuge-app/src/pages/Onboarding/EmailVerified.tsx:15]: or choose a value that's already in our theme

**Review by sophialittlejohn — APPROVED** [review]
💯

**Pull request merged**